test/extended/cli/explain.go

```diff
@@ -315,6 +315,9 @@ var (
 			field:   "consoleyamlsamples.spec",
 			pattern: `DESCRIPTION\:.*`,
 		},
+	}
+
+	specialNetworkingTypes = []explainExceptions{
```

Comment:
Not sure if the test stops on the first instance it finds - or OVN doesn't have any of these?

Reply:
ovn doesn't have any of these.